templates/Canvas/utils.tsx

```diff
@@ -83,6 +83,10 @@ export const restrictToSample: RestrictToSampleModifier = (
   rootId,
   { transform },
 ) => {
+  if (typeof window === "undefined" || typeof document === "undefined") {
+    return transform;
+  }
```

Comment:
why the workaround? (according to the comments above) this is just for Storybook, which doesn't even render the templates anymore. let's just remove this